src/fides/api/alembic/migrations/versions/9e83545ed9b6_pre_approval_webhook_cleanup.py

```diff
@@ -0,0 +1,131 @@
+"""pre_approval_webhook_cleanup
+
+Revision ID: 9e83545ed9b6
+Revises: c85a641cc92c
+Create Date: 2024-04-26 23:47:27.821607
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "9e83545ed9b6"
+down_revision = "c85a641cc92c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Most of this is just the result of running another automatic alembic migration to pick up the fact that the database is missing some items on the model.  Further, there were some new model changes added that have been matched here.